gapii/cc/spy.cpp

```diff
@@ -108,6 +108,9 @@ const uint32_t kStartMidExecutionCapture =  0xdeadbeef;
 const int32_t  kSuspendIndefinitely = -1;
 
 const uint8_t kCoreAPI = 0;
```

Comment:
Not too happy with these constants needing to match those in the API file.
At the very least please can we have a comment?